funnel/templates/js/update.js.jinja2

```diff
@@ -26,24 +26,24 @@
               <ul class="bullet-separated-list--flex mui--text-light mui--text-body2" v-if="update.number">
                 <li><a class="text-uppercase mui--text-light nounderline" :href="update.urls.view">{{ gettext('Update') }} #{{ update.number }}</a></li>
```

Comment:
Just noticed this gettext mistake. The whole phrase `Update #{{ number }}` must be in gettext, and the number itself must be localized.